metals/src/main/scala/scala/meta/internal/builds/MillBuildTool.scala

```diff
@@ -0,0 +1,65 @@
+package scala.meta.internal.builds
+import scala.meta.internal.metals.{MetalsServerConfig, UserConfiguration}
+import scala.meta.io.AbsolutePath
+import scala.util.Properties
+import java.nio.file.Files
+import java.nio.file.Path
+
+class MillBuildTool extends BuildTool {
```

Comment:
Minor, but I've noticed that SbtBuildTool is a case class while MillBuildTool and GradleBuildTool are not. Is this a choice or is it accidental?

Reply:
accidental, will fix. Thanks!